usersync/chooser.go

```diff
@@ -85,13 +85,17 @@ const (
 
 	// StatusDuplicate specifies the bidder is a duplicate or shared a syncer key with another bidder choice.
 	StatusDuplicate
+
+	// StatusBlockedByPrivacy specifies a bidder sync url is not allowed by privacy activities
+	StatusBlockedByPrivacy
 )
 
 // Privacy determines which privacy policies will be enforced for a user sync request.
 type Privacy interface {
 	GDPRAllowsHostCookie() bool
 	GDPRAllowsBidderSync(bidder string) bool
 	CCPAAllowsBidderSync(bidder string) bool
+	ActivityAllowsUserSync(bidder string) bool
```

Comment:
Should we rename to `ActivityAllowsBidderSync` to match `GDPRAllowsBidderSync` and `CCPAAllowsBidderSync`?

Reply:
Gus, activity name we execute here is `syncUser` : 
https://github.com/prebid/prebid-server/blob/57236741c7464cb500aaa9b05508460982c53382/privacy/activity.go#L19
and this is how it is in configs: 
https://github.com/prebid/prebid-server/blob/57236741c7464cb500aaa9b05508460982c53382/config/activity.go#L4

I'm open to rename it if you think `ActivityAllowsBidderSync` is better than `ActivityAllowsUserSync`.